plan/config.go

```diff
@@ -2,19 +2,26 @@ package plan
 
 import (
 	"os"
+	"strconv"
 	"strings"
+	"time"
 )
 
-// ReadConfigFromEnviron creates a Config by looking for CROSSDOCK_ environment vars
-func ReadConfigFromEnviron() Config {
-	const clientsKey = "CROSSDOCK_CLIENTS"
-	const axisKeyPrefix = "CROSSDOCK_AXIS_"
-
-	clients := strings.Split(os.Getenv(clientsKey), ",")
-	clients = trimCollection(clients)
+const callDeadlineDefault = 5
 
+// ReadConfigFromEnviron creates a Config by looking for CROSSDOCK_ environment vars
+func ReadConfigFromEnviron() *Config {
+	const (
+		callDeadlineKey = "CROSSDOCK_CALL_DEADLINE"
```

Comment:
Might be less confusing to call it TTL, since it’s a duration relative to start time. Deadline implies an absolute time.